### ISSUE TITLE
Show results of "Block Trackers" option for Vivaldi browser somewhere

### ISSUE THREAD

**Issue by arthuredelstein** [privacytests/privacytests.org#197]
The tracker blocker is disabled in Vivaldi browser. But Vivaldi does ask users on setup if they would like to enable the tracker blocker. So I am considering showing the result in that case, perhaps on a second page.

**Comment by burdges** [comment]
Yes, there is both the tracker protection and the add blocker mode.  It'll stay weaker than brave I guess, but it'd be nice to know how much weaker.
